src/components/Stepper/index.tsx

```diff
@@ -149,6 +149,10 @@ function GnoStepper<V>(props: GnoStepperProps<V>): React.ReactElement {
         validation={validate}
       >
         {(submitting, validating, ...rest) => {
+          if (submitting) {
+            return null
+          }
```

Comment:
Is there any way to render _something_ (for the transition to happen) but not the gas-fetching form?

Reply:
I haven't found a way to do this.

Reply:
Might be a good test challenge for frontend candidates 🤣